components/BcsButtons.vue

```diff
@@ -61,6 +63,10 @@ export default {
       type: Boolean,
       default: undefined
     },
+    hideUploadButton: {
+      type: Boolean,
+      default: undefined
```

Comment:
if the `type` is boolean, would a type of `false` be more appropriate?

Reply:
The BarcodeScanner one was updated, but this BcsButtons one should be too I assume.